docs/src/markdown/contributing.md

```diff
@@ -37,16 +37,16 @@ Take part in reviewing pull requests and/or reviewing direct commits.  Make sugg
 
 ## Pull Requests
 
-Pull requests are welcome, and if you plan on contributing directly to the code, there are a couple of things to be mindful of.
+Pull requests are welcome, and if you plan on contributing directly to the code, there are a couple of things you should bare in mind.
 
-Continuous integration tests on are run on all pull requests and commits via Travis CI.  When making a pull request, the tests will automatically be run, and the request must pass to be accepted.  You can (and should) run these tests before pull requesting.  If it is not possible to run these tests locally, they will be run when the pull request is made, but it is strongly suggested that requesters make an effort to verify before requesting to allow for a quick, smooth merge.
+Continuous integration tests on are run on all pull requests and commits via [Travis CI][travis].  When making a pull request, the tests will be automatically run, and the request must pass to be accepted.  You can (and should) run these tests locally before pull requesting.  If it's not possible to run them locally, they will be run when the pull request is made, but it's strongly recommend that requesters make an effort to verify before requesting to allow for a quick, smooth merge.
```

Comment:
Just something I noticed (not that you introduced):

Continuous integration tests ~~on~~ are run

Reply:
Also:

"but it's strongly recommend that" > "recommended"